includes/variables.php

```diff
@@ -1418,4 +1419,10 @@
 
 	$tsml_types_in_use = get_option('tsml_types_in_use', []);
 	if (!is_array($tsml_types_in_use)) $tsml_types_in_use = [];
-});
+};
+
+add_action('plugins_loaded', 'tsml_load_config');
```

Comment:
below, just seeing if ^ this hook will never trigger, than call it now.
- don't love the function name, glad to rename
- can move ^ into an else clause if you prefer